README.md

```diff
@@ -20,12 +20,16 @@ You are expected to have Elixir and Postgres installed.
 
     mix deps.get
     mix ecto.setup
-    mix phoenix.server
+    npm install
```

Comment:
Hm, maybe we should change line 19 to "Phoenix, NPM and Postgres"? I think this bit is part of the Phoenix installation instructions. If so, maybe it doesn't fit in the instructions for this app.